python/data/dataset/imagenet_dataset.py

```diff
@@ -22,3 +22,19 @@ def _load_anno(self, seed=None):
                 self.images.append(os.path.join(self._img_root, l[0]))
                 self.labels.append(np.int64(l[1]))
                 assert os.path.exists(self.images[-1])
+
```

Comment:
cityscapes is a new dataset.  Move Cityscapes to cityscapes_dataset.py, and rename the class to CityScapesDataSet.